src/07-projects/milestones.apib

```diff
@@ -147,6 +149,7 @@ Update a milestone.
         + id: `67e80ad8-d14f-4510-a2bd-a4c6aa578c37` (string, required)
         + due_on: `2018-01-01` (string, optional)
         + name: `Initial setup` (string, optional)
+        + description (string, optional)
```

Comment:
Q: kinda the same question here. If we want to unset a description of a milestone, we need to be able to pass `null` as a value, hence the property should be nullable.

Not passing it would equal not updating it.

Reply:
It's possible to unset the description by passing an empty string.

Reply:
> It's possible to unset the description by passing an empty string.

More correctly, the description cannot be unset, but it can be an empty string.

Reply:
IIRC, we do have endpoints that treat `''` as `null` and unset the property. Cfr the other comment as well.